src/api/view.js

```diff
@@ -75,8 +75,14 @@ export function createViewRequest({
 
 export function filterViewRequest(windowId, viewId, filters) {
   filters.map(filter => {
+    delete filter.defaultVal;
```

Comment:
the filter shall contain only following 2 params:
* `filterId`
* `parameters`

why not building an object only with those 2?

Reply:
Is this documented anywhere ? I don't want to guess this kind of stuff.

Reply:
sure it is, swagger is our source of true.
In this case: http://w101.metasfresh.com:8080/swagger-ui.html#/view-rest-controller/filterViewUsingPOST

![image](https://user-images.githubusercontent.com/1244701/61394822-b087df00-a8cc-11e9-9268-1b768196469a.png)



The only this is not obvious on swagger is that u don't have to set `caption` field.